src/web/core/src/main/java/org/geoserver/web/css/geoserver.css

```diff
@@ -1070,6 +1070,11 @@ height: 2.1em;
 margin: 1.5em 0 2.5em;
 }
 
+.tab-row-compact {
+margin: 1.5em 0 1em;
```

Comment:
Given the margin of the `page-header`, a style of `margin: -1em 0 1em;` might work better here (Or `margin: 0 0 1em;` at least) . Result:

![screen shot 2016-08-15 at 9 18 25 am](https://cloud.githubusercontent.com/assets/9271497/17670955/4383ec24-62c9-11e6-8670-f6cad7b1bb2b.png)